heron/storm/src/java/shade.conf

```diff
@@ -0,0 +1,4 @@
+rule com.google.** org.apache.storm.shaded.@0
+rule com.twitter.** org.apache.storm.shaded.@0
```

Comment:
What if user accesses some APIs in `com.twitter.heron.xxx`? Is it OK? If we shade, user will see something like `org.apache.storm.shaded.do.not.use.com.twitter.heron.xxx`, and he will not use it either intentionally or accidentally for sure.